Dockerfile.alpine

```diff
@@ -0,0 +1,19 @@
+FROM amd64/alpine:3.15 AS build
+
+RUN apk update \
+    && apk add clang build-base bash icu-libs krb5-libs \
+    libgcc libintl libssl1.1 libstdc++ zlib zlib-dev
+
+RUN mkdir -p /usr/share/dotnet && ln -s /usr/share/dotnet/dotnet /usr/bin/dotnet
```

Comment:
https://linuxize.com/post/how-to-create-symbolic-links-in-linux-using-the-ln-command/

If I tried to install on line 11 to `usr/bin/dotnet` when I tried to publish I would get permission denied error. Is there another way to do this?